src/Datagrid/ProxyQueryInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrineORMAdminBundle\Datagrid;
+
+use Doctrine\ORM\QueryBuilder;
+use Sonata\AdminBundle\Datagrid\ProxyQueryInterface as BaseProxyQueryInterface;
+
+interface ProxyQueryInterface extends BaseProxyQueryInterface
+{
+    /**
+     * @return QueryBuilder
```

Comment:
Should we add a `NEXT_MAJOR` comment indicating that this docblock must be moved to the return type declaration?

Reply:
👎 That should be clear. We need to do this for every method.

Reply:
Good. It's just the panic for the lack of the interface.

Reply:
https://github.com/sonata-project/SonataDoctrineORMAdminBundle/issues/1188